.prettierignore

```diff
@@ -0,0 +1,14 @@
+submodules
+node_modules
+build
+component-data
+manifests
+resources
+
+lib/adBlockRust*/

```

Comment:
the modern prettier supports multiple `ignore-path`.
That way you don't need to copy `.gitignore`, just use:
`--ignore-path .prettierignore --ignore-path .gitignore`